installers-conda/resources/post-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -i
```

Comment:
I noticed that this change adds all contents of this script to my bash history (which is really annoying). So you should disable doing that. [Here](https://stackoverflow.com/q/6475524/438386) are some options to achieve it.

By the way, the same is happening for the new script you added that gets the user env vars declared in `~/.bashrc`. So, you should do the same for it (either here or in another PR, up to you).

Reply:
Excellent observation! I would not have caught that. I'll fix the installers in this PR and make another one for the user env variables.